CSS-Grid-Opdracht/grid-opdracht.html

```diff
@@ -5,7 +5,23 @@
     <link rel="stylesheet" href="styles.css">
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
-<body id="test">
-Test
+<body class="container">
+<header class="class-items" id="xheader">Header</header>
+<aside class="class-items" id="xb-aside">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt facere magni
```

Comment:
Normaal doe je tekst altijd in een <p> tag. Waarschijnlijk dat je er dit keer niet aan gedacht hebt omdat we meer met de css bezig waren 😄